frontend/appflowy_tauri/src/appflowy_app/components/document/BlockSideToolbar/MenuItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { ListItem, ListItemButton, ListItemIcon, ListItemText } from '@mui/material';
+
+function MenuItem({
+  icon,
+  title,
+  onClick,
+  extra,
+  onHover,
+}: {
+  title: string;
+  icon: React.ReactNode;
+  onClick?: () => void;
+  extra?: React.ReactNode;
+  onHover?: (isHovered: boolean, event: React.MouseEvent<HTMLDivElement>) => void;
+}) {
+  return (
```

Comment:
Would it be better to wrap the parameters as a type or an interface?

I found a reference link.
https://www.technicalfeeder.com/2022/08/typescript-make-the-function-parameter-more-maintainable-by-using-interface/

Reply:
In fact, it is the same. My current way of writing is to encapsulate it into an interface